test/basic-types-test.js

```diff
@@ -47,15 +47,15 @@ test('Analyzer: boolean validator', function t(assert) {
 
   arr = [1, 0, 1, 0, 1, 0].map(mapArr);
   assert.equal(Analyzer.computeColMeta(arr)[0].type, 'INT',
-    'correctly inteprets ones and zeros as int, not booleans');
+    'Inteprets ones and zeros as int, not booleans');
```

Comment:
the only thing that could possibly be more correct is [*Inte**r**prets*](https://www.merriam-webster.com/dictionary/interpret).